gui/seedwatch.lua

```diff
@@ -0,0 +1,310 @@
+-- config ui for seedwatch
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local plugin = require('plugins.seedwatch')
+
+local PROPERTIES_HEADER = '   Quantity     Target   '
+local REFRESH_MS = 10000
+local MAX_TARGET = 2147483647
+--
+-- SeedSettings
+--
+SeedSettings = defclass(SeedSettings, widgets.Window)
+SeedSettings.ATTRS{
+    lockable=false,
+    frame={l=5, t=5, w=35, h=9},
+    data={id='', name='', quantity=0, target=0,},
+}
+
+function SeedSettings:init()
+    self:addviews{
+        widgets.Label{
+            frame={t=0, l=0},
+            text='Seed: ',
+        },
+        widgets.Label{
+            view_id='id',
+            frame={t=0, l=0},
+            visible=false,
+        },
```

Comment:
was this for debugging? it appears to overlap the previous field and is never made visible

Reply:
It was used as a passback variable, but also used for debugging.